SinaiStCatherine/SinaiNewEt002.xml

```diff
@@ -207,33 +392,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                          <q xml:lang="gez">ለዕመ ዐልብየ፡ ዝንቱመጸሀ<gap reason="lost"/> ዐልብየ፡ ተናዝዞት፡ ካልዕ</q>
                        </item>
                        <item xml:id="a6">
-                         <locus target="#110v"/>
-                         <desc type="Unclear">Note in the upper margin.</desc>
-                         <q xml:lang="gez">ሀይጣሕል፡ ዝንቱ፡ መጸሐፈ፡ ንሳዕዐ</q>
+                         <locus target="#64v"/>
+                         <desc type="Unclear">Written in the upper margin in red ink.</desc>
+                         <q xml:lang="gez">ቶማስ፡</q> <!--create person ID?-->
                        </item>
                        <item xml:id="a7">
                          <locus target="#110v"/>
-                         <desc type="Unclear">Note in the lower margin, possibly continuing <ref target="#a6"/>.</desc>
-                         <q xml:lang="gez">ዐበጊ፡ ዘወተር፡ ሀይጣሕል፡ ከተከከ<gap reason="lost"/>ክርስቶስ፡ ሀተታዐምር</q>
+                         <desc type="Unclear">Note in the upper margin, partly incomprehensible.</desc>
+                         <q xml:lang="gez">ሀይጣሕል፡ ዝንቱ፡ መጸሐፈ፡ ንሳዕዐ</q>
                        </item>
                        <item xml:id="a8">
+                         <locus target="#110v"/>
+                         <desc type="Unclear">Note in the lower margin, possibly continuing <ref target="#a6"/>, partly incomprehensible.</desc>
+                         <q xml:lang="gez">ዐበጊ፡ በወተር፡ ሀይጣሕል፡ ከተከከ<gap reason="lost"/>ክርስቶስ፡ ሀተታዐምር</q>
+                       </item>
+                       <item xml:id="a9">
                          <locus target="#111r"/>
                          <desc type="Supplication">Supplication in the upper and right margin.</desc>
                          <q xml:lang="gez">በጸሎትከተዘከርኒ፡ በሀጢዐት<gap reason="lost"/> ገብረክርስ</q>
                        </item>
+                       
+                       <item xml:id="e1">
+                         <desc>The manuscript contains indications for liturgical readings throughout, both in the upper margin and in the body of the text. Red numerals indicating 
+                         sections are written to the left of the column.</desc>
+                       </item>
                       </list>
                   </additions>
 
                   <bindingDesc>
                      <binding contemporary="non-Ethiopian" xml:id="binding">
-                        <decoNote xml:id="b1">Islamic binding, boards of paper or cardboard covered with blind-tooled leather, with flap.</decoNote>
-                        <decoNote xml:id="b6" type="bindingMaterial">
+                        <decoNote xml:id="b1">Islamic binding, boards of paper or cardboard covered with blind-tooled leather, with flap. A small leather strip is inserted into the hole in the flap.</decoNote>
+                       <decoNote xml:id="b2" type="Cover">Blind-tooled brown leather cover.</decoNote>
```

Comment:
If you look closer, you will see that the multiple rosettes have the same decorativ motif inside, smth like bird (cock?)